components/withNavigation.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { useNavigation } from '@react-navigation/native'; // not sure package name
+
+export const withNavigation = (Component) => {
```

Comment:
question: is there a benefit to create a HOC instead of using useNavigation() directly in the components?